### ISSUE TITLE
Failed to locate platform-specific led_control module.

### ISSUE THREAD

**Issue by lguohan** [Azure/sonic-platform-daemons#6]
happens on 7050-qx32 platform

running inside docker, plugin is in directory /usr/share/sonic/platform/

**Comment by lguohan** [comment]
```
Jun 19 09:10:45.0 str-a7050-acs-2 INFO ledd: Starting up...
Jun 19 09:10:52.0 str-a7050-acs-2 INFO ledd: Starting up...
Jun 19 09:10:53.0 str-a7050-acs-2 INFO ledd: Failed to locate platform-specific led_control module. Exiting...
```

**Comment by lguohan** [comment]
fixed in #7
